dotCMS/src/main/java/com/dotmarketing/portlets/browser/ajax/BrowserAjax.java

```diff
@@ -1607,7 +1610,38 @@ public boolean deleteAsset(String inode) throws Exception
         return result;
     }
 
-    public Map<String, Object> changeAssetMenuOrder (String inode, int newValue) throws ActionException, DotDataException {
+	/**
+	 * Verifies if a page is being used as a detail page for any content type
+	 * @return
+	 * @throws DotDataException
+	 * @throws LanguageException
+	 */
+	public Map<String, Object> validateRelatedContentType(String inode)
+			throws DotDataException, LanguageException, DotSecurityException {
+
+		Map<String, Object> result = new HashMap<String, Object>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)